DuckDuckGo/NetworkProtection/NetworkExtensionTargets/NetworkExtensionTargets/MacPacketTunnelProvider.swift

```diff
@@ -397,13 +385,14 @@ final class MacPacketTunnelProvider: PacketTunnelProvider {
 #if NETP_SYSTEM_EXTENSION
         "\(Bundle.main.bundleIdentifier!).authToken"
 #else
-        NetworkProtectionKeychainTokenStore.Defaults.tokenStoreService
+        "com.duckduckgo.networkprotection.authToken"
```

Comment:
Consider adding a constant here.

Reply:
I transformed tokenServiceName in a let so that's now a constant, there was no need to be a var calculated at runtime.
I didn't move the string somewhere else (like in the previous implementation) because is only used there.